test/unit/solver/mc/test_triangular.cpp

```diff
@@ -97,3 +156,24 @@ TYPED_TEST(TriangularSolverLocalTest, Correctness) {
     }
   }
 }
+
+TYPED_TEST(TriangularSolverDistributedTest, Correctness) {
+  SizeType m, n, mb, nb;
+
+  for (const auto& comm_grid : {this->commGrids()[0]}) {
+    for (auto diag : blas_diags) {
+      for (auto op : blas_ops) {
+        for (auto side : blas_sides) {
+          for (auto uplo : blas_uplos) {
```

Comment:
I would reorder the loops to have the same order as in the function argument list.